### DISCUSSION TITLE
Close to System Tray

### DISCUSSION THREAD

**Discussion by PLCNoob** [snappysnappydog/Simple-IP-Config#163] [Ideas]
Hi, Thank you for building this useful tool!
I think it would be nice to have a setting that, when enabled, sends SimpleIPConfig to the system tray instead of closing it when you hit the X.
For my workflow, I prefer to have SimpleIPConfig running at all times. I want to minimize it to the task bar, and close it to the system tray. That way, I never have to re-launch the program whenever I close it. Of course, with this feature you'd still be able to fully close the program by right clicking on the system tray icon.

**Comment by KurtisLiggett** [comment]
@PLCNoob , you can nearly accomplish this now by checking the setting "Minimize to the system tray".
The only difference is you would hit the minimize button ( - ) to send the program to the system tray instead of the close button ( x ). Then you can click the tray icon to restore.
Does that work for you?

Note:
There's also a hidden feature: no matter which settings you have applied, you can always do a single click on the system tray icon to hide/show the app.

**Reply by AuWiMo** [reply]
I would like to second this request for the same reasons. 
The reason that minimize to system tray is not as helpful is that the natural inclination is to click the close button and then I have to open it again later.

Great tool by the way, I love it!

**Comment by KurtisLiggett** [comment]
@PLCNoob @AuWiMo 
I don't mind making this change. I don't think we need both options though. Do you think we should just change the "Minimize to the system tray" to "Close to the system tray"? So the close button would send the app to the tray but the minimize button always just minimizes to the taskbar as normal?

**Reply by PLCNoob** [reply]
I don't think it matters haha. At this point I've gotten used to it. I do think that that is a little more consistent with how other system tray tools work.
I've also discovered that SimpleIPConfig does not recognize when I connect a USB C Ethernet Adapter, so I end up having to restart the program pretty frequently. With that workflow, it's actually useful for the close button to kill the program.

Now, if you were to add a "Refresh Adapters" button, then I would definitely want the close button to minimize to system tray so that I don't accidentally close it.

**Reply by AuWiMo** [reply]
Thank you, I feel like that would be a wise choice. I do think that "close to system tray" is the standard. I have never seen "minimize to system tray before"

**Comment by KurtisLiggett** [comment]
Thanks to you both for the feedback. I added it to the [project roadmap](https://github.com/KurtisLiggett/Simple-IP-Config/discussions/165) to include in the next release!

**Comment by KurtisLiggett** [comment]
@PLCNoob Thanks for pointing out the adapter discovery issue. It turns out I broke the refresh button at some point and never noticed. In the next release you should be able to click the refresh button to show new devices. #167

**Reply by PLCNoob** [reply]
Lol Idk how I never noticed that button. Or maybe I did and it didn't work.
Either way, thanks again! We love responsive devs.

**Comment by KurtisLiggett** [comment]
Now available with release v2.9.7 - [Download Here](https://github.com/KurtisLiggett/Simple-IP-Config/releases/tag/2.9.7)
